website/docusaurus.config.js

```diff
@@ -420,7 +420,15 @@ const config = {
             label: 'Documentation',
           },
           { to: '/features', label: 'Features', position: 'left' },
-          { to: '/downloads', label: 'Downloads', position: 'left' },
+          {
+            type: 'custom-telemetryLink',
+            position: 'left',
+            children: 'Downloads',
+            className: 'navbar__item navbar__link',
```

Comment:
question: why do we need to add children and className which are not added to the previous items ?

Reply:
my point is that here, I should not add any other customization

if I need to add a specific className, then it's the overriden MDX component that should add this className before sending it to the TelemetryLink


docusaurus.config.js: only the data about the events (no className, no children)

customization of the navbaritem: here it does all the logic handling the existing param and if missing, add any class/attributes about navbar item

TelemetryLink: do only the telemetry stuff